readme.md

```diff
@@ -37,13 +37,13 @@ The Authentication argument is optional. You must first create a user with `gith
     github readme.md
 
 #### Adding Authentication
-To use authenticated requests you must first [create a github](https://help.github.com/articles/creating-an-access-token-for-command-line-use/) api token on your github account.
+To use authenticated requests you must first [create a github](https://help.github.com/articles/creating-an-access-token-for-command-line-use/) api token on your github account. You do not need to provide any api scopes for the token unless you plan on accessing private repositories.
 
-    github auth <user> <token> [-D <username>]
+    github auth <user> [<api token> | -D]
```

Comment:
I'd argue lowercase d because 1) it's easier to type, 2), `git branch -d <name>` is a thing